### ISSUE TITLE
Gaming

### ISSUE THREAD

**Issue by chris-y** [fnic-lemmy/lemmy-directory#11]
I know this was out there somewhere, even bookmarked it's taken me ages to locate. 

List of specific games communities:
https://kbin.cafe/m/gaming@beehaw.org/t/124541

**Comment by M4rtineski** [comment]
For other categories I'm also supplementing myself with this list: https://lemmy.world/post/2216085

Not really related but thought it would be nice to have both lists in one place.

**Comment by chris-y** [comment]
There's also this which might be useful to refer to (instances not communities):
https://kbin.social/m/specialized_instances/t/186667

**Comment by M4rtineski** [comment]
I think it's a good idea to link the list at the end of the directory but I don't want to organise instances so I won't take it any further.

**Comment by chris-y** [comment]
I mentioned it more in case it helped you find communities for the other categories. It's a useful one to link though. Not sure if the author is still active.

**Comment by M4rtineski** [comment]
Sigh, let me think about that once we're done with the directory. xD

**Comment by chris-y** [comment]
We should be able to submodule it, then the maintenance is from upstream - we just have a copy which we can stick in a separate post.
Something for later!

**Comment by M4rtineski** [comment]
During preparing a list of communities I noticed how many communities are on instance themed around their topic and realised that it may indeed be useful to have a simple organised list of instances. Though I would need to look into instance pickers to see if such list is even needed.

**Comment by chris-y** [comment]
touhou@lemmy.world
Apparently this is a series of shoot-em-ups.

**Comment by M4rtineski** [comment]
Touhou is already there at the category 5.2. and entry 41.

**Comment by chris-y** [comment]
https://feddit.uk/post/2800598

Looks like ps5@lemmy.world moved/merged since a year ago, so needs removing from the directory.

**Comment by chris-y** [comment]
sonicthehedgehog@lemmy.world

Might fall under "franchise" but most of the video game discussion communities are like this so I'd be inclined to put it under Games.

**Comment by chris-y** [comment]
Not sure if any genres will turn up we don't already know about: https://feddit.org/post/5013472

**Comment by M4rtineski** [comment]
Oh shoot, If you let me know beforehand I'd have quickly updated it before sharing. xd

**Comment by chris-y** [comment]
Yeah, there's broken instances on there I didn't realise 🙁

**Comment by chris-y** [comment]
Another genres list here: https://lemmy.dbzer0.com/post/42102304

**Comment by chris-y** [comment]
Yeah, it appears to have been deleted. I can't remember which community it was in to look for it.

**Comment by M4rtineski** [comment]
I got an admin to read the post for me. lol

!adventuregames@lemm.ee (also has puzzle and interactive fiction)
    !arcaderacing@lemm.ee / !SimRacing@lemmy.zip
    !automationgames@lemmy.zip (replaces dead feddit.de community)
    !citybuilders@sh.itjust.works
    !cozygames@lemmy.world (replaces dead feddit.de community)
    !crpg@lemmy.world
    !dailygames@lemmy.zip (for quick once-a-day web games like Wordle)
    !incremental_games@incremental.social
    !jrpg@lemmy.zip
    !lifesimulation@lemmy.world
    !otomegames@ani.social
    !rhythmgames@lemm.ee
    !shmups@lemmus.org
    !soulslike@lemmy.zip
    !tycoon@lemmy.world
    !visualnovels@ani.social

**Comment by chris-y** [comment]
Suspect this is an updated version of the same: 
https://lemmy.zip/post/36589987

````````
List more in the comments! I'll add them if they are a community for a video game genre, and have had at least one post in the last month. Crossposted to !games@lemmy.world [here](https://lemmyverse.link/lemmy.zip/post/36589978).

- !adventuregames@lemm.ee (also has puzzle and interactive fiction)
- !arcaderacing@lemm.ee
- !automationgames@lemmy.zip (replaces dead feddit.de community)
- !citybuilders@sh.itjust.works
- !cozygames@lemmy.world (replaces dead feddit.de community)
- !crpg@lemmy.world
- !dailygames@lemmy.zip (for quick once-a-day web games like Wordle) 
- !fgc@lemmy.dbzer0.com and !fgc@lemmy.world
- !incremental_games@incremental.social (having federation troubles, but as someone with an incremental.social account I can guarantee we are active)
- !jrpg@lemmy.zip and !jrpg@lemmy.world
- !lifesimulation@lemmy.world
- !metroidvanias@infosec.pub
- !otomegames@ani.social
- !rhythmgames@lemm.ee 
- !roguelikes@lemmy.world
- !shmups@lemmus.org
- !simrallyracing@lemmy.world and !simracing@lemmy.zip
- !soulslike@lemmy.zip 
- !strategy@lemmy.world, !strategygames@sopuli.xyz, !turnbasedstrategy@lemmy.world, !turn_based_tactics@lemmy.zip
- !tycoon@lemmy.world
- !visualnovels@ani.social

I also found some genres with dead communities. Maybe someone would like to revive one, or make a new one (preferably on a smaller instance than lemmy.world)?

- !horrorgaming@lemmy.world
- !platformer_games@sh.itjust.works
- !puzzlegames@lemm.ee (yes, !adventuregames has some, whether puzzles actually belong with adventure games is another conversation. I don't think so but I'm not an expert on either genre)
- !realtimestrategy@lemmy.world and !rts@reddthat.com
- !survivalgames@lemm.ee

On my search for communities to put here I found some genres seem to be wholly missing. Let me know if I missed something!

- MMOs
- open world/sandbox
- RPGs that don't really fit in CRPG or JRPG
- shooters
- simulation in general
- sports
- stealth
```
